docs/contracts/visibility-and-getters.rst

```diff
@@ -158,13 +157,13 @@ to write a function, for example:
 
     // Getter function generated by the compiler
     /*
-    function myArray(uint i) returns (uint) {
+    function myArray(uint i) public view returns (uint) {
```

Comment:
Weird, this should have caused a test failure. Thanks!

Reply:
Ah, this contract is not extracted properly because the indentation is off.

Reply:
Fixed with https://github.com/ethereum/solidity/pull/7564.